04-kogito-travel-agency/pom.xml

```diff
@@ -48,6 +48,12 @@
       <groupId>org.kie.kogito</groupId>
       <artifactId>kogito-events-reactive-messaging-addon</artifactId>
       <version>${kogito.version}</version>
+      <exclusions>
+        <exclusion>
+          <groupId>io.smallrye.reactive</groupId>
+          <artifactId>smallrye-reactive-messaging-provider-1.0</artifactId>
```

Comment:
this is no longer needed as we already upgraded to quarkus 0.23 that uses kogito 0.4